eclipse-pass/pass-docker
# Update demo user data

Fixes https://github.com/eclipse-pass/main/issues/517
# Changes
* Update demo user data to match ldap data in order to fix login integration w/user service
  * This data mismatch caused incorrect "shib" headers to be sent, causing duplicate users to be created by the user service
* Remove old Harvard stuff

# Testing

New Docker images has not been published yet. To test:

* Against code on `main` branch, run `docker compose up -d --no-build --pull always --quiet-pull` to run the demo stack
* Observe that logins fail with demo users (e.g. `nih-user`)
* Bring stack down, deleting data: `docker compose down -v`
* Pull code for this PR. Since only bootstrap data is changed, nothing needs to be rebuilt
* Bring demo stack up again: `docker compose up -d --no-build`
* Logins should now work
